tokio/src/runtime/time/mod.rs

```diff
@@ -406,5 +442,24 @@ impl fmt::Debug for Inner {
     }
 }
 
+mod rand {
+    use crate::util::rand::FastRand;
+    use std::cell::Cell;
+
+    // Used by `TimerEntry`.
+    pub(crate) fn thread_rng_n(n: u32) -> u32 {
+        thread_local! {
+            static THREAD_RNG: Cell<FastRand> = Cell::new(FastRand::new());
+        }
```

Comment:
Please do not create new thread locals. Some platforms have small limits on the number of thread locals that each process can have.

There should already be a random number generator in the runtime context somewhere.

Reply:
I used the `context::thread_rng_n` instead. But this introduces unexpected additional `time` features. 

If we are not allowed to access 'context:: thread_rng'n' in the `time` feature, then our other option may be to use `std::thread::current().id().as_u64()`. However, the latter is unstable. Maybe the hash value of thread id can address this problem.